docs/modules/ROOT/pages/references/decision-deployment-strategy.adoc

```diff
@@ -0,0 +1,160 @@
+= Decision: Deployment Strategy For Postgresql Standalone
```

Comment:
Although this specific instance is about the postgresql instance, I feel like will have to go through this decision for every new service. So this may be something to considere for a template for future services?